src/components/characters/Characters.js

```diff
@@ -3,15 +3,17 @@ import PropTypes from 'prop-types';
 import Character from '../characters/Character';
 import { getCharacters } from '../../services/rickAndMortyApi';
 import { withPaging } from '../paging/Paging';
+import { withFetch } from '../fetch/Fetch';
 
-export default class Characters extends PureComponent {
+class Characters extends PureComponent {
   static propTypes = {
     page: PropTypes.number,
-    updateTotalPages: PropTypes.func.isRequired
+    updateTotalPages: PropTypes.func
```

Comment:
```suggestion
    updatedTotalPages: PropTypes.func
```
Looks like 'updatedTotalPages' was the prop passed from Paging -- double check naming on parent and child components.